.gitignore

```diff
@@ -1,3 +1,6 @@
+# macOS
+.DS_Store
```

Comment:
You'll need to remove this, delete the file, then re-add this (otherwise it will ignore that you deleted the file).